src/helpers.ts

```diff
@@ -58,10 +58,14 @@ export function concat(existingText: string, newText: ppa.CellSlice): string {
  */
 export function convertVscToGatherCell(cell: vscode.NotebookCell): ppa.Cell | undefined {
   let code = '';
-  if (cell.document) {
-    code = cell.document.getText();
-  } else {
-    code = (<any>cell).code as string;
+  try {
+    if (cell.document) {
+      code = cell.document.getText();
+    } else {
+      code = (cell as any).code as string;
```

Comment:
This still seems like a problem waiting to happen. Somebody in the jupyterExtension may not realize this dependency in the future as it's not actually part of the API.

We can log an issue to fix later though.

Reply:
If we deprecate the webview notebooks and move IW over to the proposed API, this wouldn't be necessary.